src/data/__tests__/allergy.test.js

```diff
@@ -1,6 +1,7 @@
 /* global test, describe, it, expect */
 'use strict'
-import allergies from '../../../files.js'
+import {allergiesFilePath} from '../../../files.js'
```

Comment:
can you replace it with require?

Reply:
const {allergiesFilePath} = require(''../../../files'')

Reply:
change it in other files

Reply:
Yes I can, but it makes no difference. 
I’ve made these changes in most of the tests. Some are left, because I’ll need to change variable names in file.js.

Reply:
import it's es6,  require it's es5. If we will leave tests with import it doesn't  work because  it's need to generate code from es6 into es5. This work is doing babel. I have fixed this by editing babel config. But if we will using es5 and es6 together and disabling generating from es6 to es5 we will get error.